pkg/provider/aws.go

```diff
@@ -204,18 +212,20 @@ func (aws *AWSProvider) Flush() error {
 }
 
 func (prov *AWSProvider) Decommision(node *v1.Node) error {
-	sess, err := session.NewSession(&aws.Config{
-		Region: aws.String(prov.Region()),
-	})
+	cfg, err := awsConfig.LoadDefaultConfig(context.TODO())
```

Comment:
Let's set `ctx := context.TODO()` and pass them to all the funcs here instead of calling it again and again

Reply:
Then it's probably better to use `context.Background()` and add it to the provider struct.